### PR TITLE
feat(remainingUIMigration): Completed the remaining UI Migration

### PR DESCRIPTION
**Pull request by sid-bagwe**
### Functionality:
Completed the remaining UI: create new team, update team and create new organization pages.

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start
Go to the my organization page check the create team and update team pop-up modals

### CODE REVIEW

**Review by viv9k — APPROVED** [review]
Approved

File: src/app/body/create-new-organization/create-new-organization.component.html [hunk 1]
```diff
@@ -1,70 +1,67 @@
...
-                <div class="form-group row pt-2">
-                    <label for="input" class="col-sm-3 col-form-label">Organization Email<span id="important" class="pl-1">*</span></label>
-                    <div class="col-sm-8">
-                        <input type="text" class="input" [(ngModel)]="orgEmail" [ngModelOptions]="{standalone: true}">
+                    <div class="form-group row pt-2">
+                        <div class="col-sm-8">
+                            <input type="text" class="form-control" placeholder="Organization Email*" [(ngModel)]="orgEmail" [ngModelOptions]="{standalone: true}">
+                        </div>
                     </div>
+                    <!-- <div class="form-group row pt-2">
```
Comment (k0malSharma) [inline src/app/body/create-new-organization/create-new-organization.component.html:35]: Remove the comment

**Review by k0malSharma — CHANGES_REQUESTED** [review]
Balance the col in html files so that they add up to 12

**Review by k0malSharma — APPROVED** [review]
Approved

**Pull request merged**